### PR TITLE
clarifications added for Slurm account

### PR DESCRIPTION
**Pull request by dsajdak**
Also reorganized slurm batch script examples to lead with cluster, partition, qos directives

### CODE REVIEW

File: pages/hpc/jobs.md [hunk 1]
```diff
@@ -170,11 +170,24 @@ supervision or interaction. Batch jobs are commonly used for applications that
...
+#SBATCH --partition=partition-name
+#SBATCH --qos=qos-name
+
+# 	Tell Slurm which account to run this job under 
+#   If not specified, your default account will be used  
+#   Use the `slimits` command to see what accounts you have access to
+
+#SBATCH --account=SlurmAccountName  
+
 #
```
Comment (KevinVanSlyke) [inline pages/hpc/jobs.md:191]: Empty comment line can be removed

File: pages/hpc/jobs.md [hunk 2]
```diff
@@ -170,11 +170,24 @@ supervision or interaction. Batch jobs are commonly used for applications that
...
 #!/bin/bash -l
+
+# 	Tell Slurm which cluster, partition and qos to use to schedule this job
+
+#SBATCH --cluster=cluster-name
+#SBATCH --partition=partition-name
+#SBATCH --qos=qos-name
+
+# 	Tell Slurm which account to run this job under 
+#   If not specified, your default account will be used
```
Comment (KevinVanSlyke) [inline pages/hpc/jobs.md:186]: The format of white space between line 185-186 isn't the same in the text document, but they show identically on Mkdocs. There's a few other comments in this code block that differ in the same way.

**Review by KevinVanSlyke — APPROVED** [review]
Couple small notes on the first large code block.

File: pages/hpc/jobs.md (hunk 2 above)
Comment (dsajdak) [reply pages/hpc/jobs.md:186]: Yeah this is due to 2 different people writing these.  I've fixed to align with original author's formatting

**Pull request merged**